client/cypress/e2e/earn.cy.ts

```diff
@@ -184,5 +187,55 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight, isDes
             .should('have.text', '1 GLM');
         });
     });
+
+    it('Wallet connected: Lock 1000 GLM + move epoch', () => {
+      connectWallet();
+
+      cy.get('[data-test=BoxGlmLock__Section--current__DoubleValue__primary]')
+        .invoke('text')
+        .then(text => {
+          const amountToLock = 1000;
+          const lockedGlms = parseInt(text.replace(/\u200a/g, ''), 10);
+
+          cy.get('[data-test=BoxGlmLock__Button]').click();
+          cy.get('[data-test=InputsCryptoFiat__InputText--crypto]').clear().type(`${amountToLock}`);
+          cy.get('[data-test=GlmLockTabs__Button]').should('have.text', 'Lock');
+          cy.get('[data-test=GlmLockTabs__Button]').click();
+          cy.get('[data-test=GlmLockTabs__Button]').should('have.text', 'Waiting for confirmation');
+          cy.confirmMetamaskPermissionToSpend({
+            spendLimit: '99999999999999999999',
+          });
+          cy.get('[data-test=GlmLockTabs__Button]', { timeout: 180000 }).should(
+            'have.text',
+            'Close',
+          );
+          cy.get('[data-test=GlmLockNotification--success]').should('be.visible');
+          cy.get('[data-test=GlmLockTabs__Button]').click();
+          cy.wait(5000);
+          cy.window().then(async win => {
+            await win.mutateAsyncMoveEpoch();
+            cy.wait(5000);
+            await axios.post(`${env.serverEndpoint}/snapshots/pending`);
```

Comment:
@mike-code: can we make `serverEndpoint` with trailing forward slash, so it's unified with the same env we set for the client?

@jmikolajczyk: if this env was never visible to the CY the same must go for JSON RPC API endpoint which may explain why you initial solution for time shifting didn't work. Wagmi config was initialized w/o our envs, falling back to defaults. Can you please coordinate and check if that works after adding an env? It would be better to have time shifting in CY directly instead of current hack from the app itself.

Reply:
@aziolek I started testing time shifting on gitlab and `VITE_JSON_RPC_API` env variable was set there -> https://gitlab.com/golemfoundation/governance/octant/-/merge_requests/952/diffs (problem existed there) but I will check it again as you proposed.

Reply:
@aziolek you may adjust it [here](https://github.com/golemfoundation/octant/pull/81/commits/7d342658c929ad06ba042eab7e34404ed4713f22#diff-ec4d64c3934019181d40d274a7c9004907161e8abd658e7d371e15a816ecbe71R93) as needed